src/test/java/com/commercetools/sync/inventories/InventorySyncTest.java

```diff
@@ -528,7 +433,6 @@ private InventorySyncOptions getInventorySyncOptions(int batchSize, boolean ensu
         return InventorySyncOptionsBuilder.of(mock(SphereClient.class))
```

Comment:
Param `allowUuid` in the method `getInventorySyncOptions` is not used and should be removed

Reply:
thanks for noticing: 
eeb03f8